src/openrct2/Game.cpp

```diff
@@ -269,7 +270,7 @@ void update_palette_effects()
         if (g1 != nullptr)
         {
             uint8_t* vs = &g1->offset[j * 3];
-            uint8_t* vd = &gGamePalette[243 * 4];
+            uint8_t* vd = &gGamePalette[PALETTE_INDEX_243 * 4];
```

Comment:
243 == 0xF3. Isn't this just PALETTE_OFFSET_REMAP_PRIMARY?

Reply:
Yes, but as I was unsure how it was used I decided to leave it as 243.